### PR TITLE
Deleted cPath::BlockTypeIsFence

### PR DESCRIPTION
**Pull request by LogicParrot**
followup to #2822

### CODE REVIEW

**Comment by LogicParrot** [comment]
@SphinxC0re

**Comment by LogicParrot** [comment]
Tested, lgtm.

File: src/Defines.h [hunk 1]
```diff
@@ -481,12 +481,19 @@ inline bool IsBlockFence(BLOCKTYPE a_BlockType)
...
 		case E_BLOCK_SPRUCE_FENCE_GATE:
+		case E_BLOCK_BIRCH_FENCE_GATE:
+		case E_BLOCK_JUNGLE_FENCE_GATE:
+		case E_BLOCK_DARK_OAK_FENCE_GATE:
+		case E_BLOCK_ACACIA_FENCE_GATE:
+		case E_BLOCK_SPRUCE_FENCE:
+		case E_BLOCK_BIRCH_FENCE:
+		case E_BLOCK_JUNGLE_FENCE:
+		case E_BLOCK_DARK_OAK_FENCE:
 		case E_BLOCK_ACACIA_FENCE:
```
Comment (bearbin) [inline src/Defines.h]: What is up with the ordering in this block? How is it meant to be ordered?

**Comment by bearbin** [comment]
Apart from the case ordering, this looks good (if it's tested and works).

The mac builder is down (I've contacted @SphinxC0re) so I wouldn't worry about any fail there.

**Comment by SphinxC0re** [comment]
Mac slave is up and running again. Builds should start soon

**Comment by LogicParrot** [comment]
Reordered and rebased.

**Comment by SphinxC0re** [comment]
OS X build worked

**Comment by jammet** [comment]
You changed your name, I think. Hope I can get used to that quick. :)

**Comment by LogicParrot** [comment]
@jammet whitelist me ;)

**Comment by jammet** [comment]
Of course :).

**Pull request merged**